database/dynamodb.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"log"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+
+)
+
+
+type AWSCredentials struct{
+	AccessKey string
+	SecretKey string
+	Region string
+}
+
+var db *dynamodb.DynamoDB
+
+func getAWSCredentials() *AWSCredentials{
+
+	awsCredentials := new(AWSCredentials)
+	awsCredentials.Region = utils.GoDotEnvVariable("AWS_REGION")
+	awsCredentials.AccessKey = utils.GoDotEnvVariable(("AWS_ACCESS_KEY"))
+	awsCredentials.SecretKey = utils.GoDotEnvVariable(("AWS_SECRET_KEY"))
+	
+	return awsCredentials
+}
+
+func createDynamoDB() *dynamodb.DynamoDB {
+	awsCredentials := getAWSCredentials()
+
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String(awsCredentials.Region),
+		Credentials: credentials.NewStaticCredentials(awsCredentials.AccessKey,awsCredentials.SecretKey,""),
+	})
+
+	if err != nil {
+		log.Println("Error while creating a session")
```

Comment:
so if there is an error instantiating a session do we just log and continue? isn't that a hard failure? sess being null? won't that throw a nil point dereference?

Reply:
+1
If there is no connection to DB, what is the use in proceeding forward, where DB operations are done?

Reply:
I'm thinking to use log.Panic() so that the program doesn't get end abruptly.
Any thoughts on this? Not `log.Fatal()` as it calls os.Exit(1)